ui/admin/tests/e2e/tests/delete-resources.spec.js

```diff
@@ -0,0 +1,197 @@
+  deleteOrgCli,
+} = require('../helpers/boundary-cli');
+
+const {
+  deleteResource,
+  removeAuthMethodAsPrimary,
+} = require('../helpers/boundary-ui');
+const { checkVaultCli } = require('../helpers/vault-cli');
+const { execSync } = require('child_process');
+
+const secretPolicyName = 'kv-policy';
+const boundaryPolicyName = 'boundary-controller';
+
+test.use({ storageState: authenticatedState });
+
+test.beforeAll(async () => {
+  await checkEnv([
+    'VAULT_ADDR',
+    'VAULT_TOKEN',
+    'E2E_VAULT_ADDR',
+    'E2E_TARGET_ADDRESS',
+    'E2E_SSH_USER',
+    'E2E_SSH_KEY_PATH',
+    'E2E_AWS_ACCESS_KEY_ID',
+    'E2E_AWS_SECRET_ACCESS_KEY',
+  ]);
+  await checkBoundaryCli();
+  await checkVaultCli();
+});
+
+test.beforeEach(async () => {
+  execSync(`vault policy delete ${secretPolicyName}`);
+  execSync(`vault policy delete ${boundaryPolicyName}`);
+  await authenticateBoundaryCli();
+});
+
+test('Verify resources can be deleted', async ({ page }) => {
+  let orgId;
+  try {
+    // Create boundary resources using CLI
+    orgId = await createNewOrgCli();
+    let projectId = await createNewProjectCli(orgId);
+    let workerId = await createNewControllerLedWorkerCli();
+    let authMethodId = await createNewPasswordAuthMethodCli(orgId);
+    await makeAuthMethodPrimary(orgId, authMethodId);
+    let passwordAccountId = await createNewPasswordAccountCli(authMethodId);
+    let projectScopeRoleId = await createNewRoleCli(projectId);
+    let orgScopeRoleId = await createNewRoleCli(orgId);
+    let globalScopeRoleId = await createNewRoleCli('global');
+    let groupId = await createNewGroupCli(orgId);
+    let userId = await createNewUserCli(orgId);
+    let staticHostCatalogId = await createNewStaticHostCatalogCli(projectId);
+    let dynamicAwsHostCatalogId = await createDynamicAWSHostCatalogCli(
+      projectId
+    );
+    let staticHost = await createNewStaticHostCli(staticHostCatalogId);
+    let staticHostSet = await createNewHostSetCli(staticHostCatalogId);
+    let staticCredentialStoreId = await createNewStaticCredentialStoreCli(
+      projectId
+    );
+    let vaultCredentialStoreId = await createNewVaultCredentialStoreCli(
+      projectId,
+      secretPolicyName,
+      boundaryPolicyName
+    );
+    let usernamePasswordCredential =
+      await createNewUsernamePasswordCredentialCli(staticCredentialStoreId);
+    let tcpTargetId = await createNewTcpTarget(projectId);
+    let sshTargetId = await createNewSshTarget(projectId);
+
+    // Delete SSH target
+    await page.goto(`/scopes/${projectId}/targets/${sshTargetId}`);
+    await deleteResource(page);
+
+    // Delete TCP target
+    await page.goto(`/scopes/${projectId}/targets/${tcpTargetId}`);
+    await deleteResource(page);
+
+    // Delete username-password credentials
+    await page.goto(
+      `/scopes/${projectId}/credential-stores/${vaultCredentialStoreId}/credentials/${usernamePasswordCredential}`
+    );
+    await deleteResource(page);
+
```

Comment:
Do we need to add the other credential types in here?

Static Credential Store
- User Password Credential
- Key Pair Credential
- JSON Credential

Vault Credential Store
- User Password Credential
- Key Pair Credential

Reply:
We do. Out of scope if this PR and ticket.